easypost/beta/user.py

```diff
@@ -0,0 +1,51 @@
+from typing import (
+    Any,
+    Dict,
+    List,
+    Optional,
+)
+
+from easypost.easypost_object import convert_to_easypost_object
+from easypost.error import Error
+from easypost.requestor import (
+    RequestMethod,
+    Requestor,
+)
+from easypost.resource import Resource
+
+
+class User(Resource):
+    @classmethod
+    def retrieve_all_children(cls, api_key: Optional[str] = None, **params) -> Dict[str, Any]:
+        """Retrieve a list of children from the API."""
```

Comment:
```suggestion
        """Retrieve a list of paginated children from the API."""
```

Reply:
+1, calling this out is important as it's the differentiating factor of this new endpoint/function and defining this well will lead to greater adoption for people looking for a solution to the current problem of all children being too large for some users

Reply:
For the "retrieve all" methods, we typically call it `.all()`, but that would get confused with "retrieve all users", so we could do `all_children()`?

Reply:
Resolved